en-US/firefox/campaign-trailhead.lang

```diff
@@ -0,0 +1,64 @@
+## NOTE: Demo page at https://www-dev.allizom.org/firefox/campaign/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/campaign/
+
+
+# HTML page title prefix
+;Download Firefox
+Download Firefox
+
+
+# HTML page title
+;Free Web Browser
+Free Web Browser
+
+
+# HTML page description
+;Download Mozilla Firefox, a free Web browser. Firefox is created by a global non-profit dedicated to putting individuals in control online. Get Firefox for Windows, macOS, Linux, Android and iOS today!
```

Comment:
Should be lower case `web`.

Reply:
Submitted https://github.com/mozilla/bedrock/pull/7343 to fix this in the template as well